predicate/examples/list_nodes.py

```diff
@@ -0,0 +1,17 @@
+from solver.ast import StringTuple
+from solver.teleport import Node, Policy, Resource, Rules
+
+
+class Teleport:
+    p = Policy(
+        name="list_nodes",
+        loud=False,
+        allow=Rules(
+            Resource(
+                (Resource.namespace == "default")
```

Comment:
@xacrimon lets remove namespaces for now from Predicate, they are not used, there is only one namespace in all resources today - default one.